ebean-core/src/main/java/io/ebeaninternal/server/core/PersistRequestBean.java

```diff
@@ -932,16 +932,16 @@ private void logSummaryMessage() {
     String name = beanDescriptor.name();
     switch (type) {
       case INSERT:
-        transaction.logSummary("Inserted [" + name + "] [" + (idValue == null ? "" : idValue) + draft);
+        transaction.logSummary("Inserted [" , name , "] [" , (idValue == null ? "" : idValue.toString()) , draft);
```

Comment:
Maybe change logger signature to `logSummary(Object... msg)`

Reply:
Yes agreed. It's starting to tell me it _could even_ be `logSummary(String msgFormat, Object... args)` ...